charts/hono/templates/artemis/artemis-deployment.yaml

```diff
@@ -53,14 +54,13 @@ spec:
           {{- . | toYaml | nindent 10 }}
         {{- end }}
         livenessProbe:
-          initialDelaySeconds: 40
+          initialDelaySeconds: {{ .Values.amqpMessagingNetworkExample.broker.artemis.livenessProbeInitialDelaySeconds }}
```

Comment:
`.Values.amqpMessagingNetworkExample.broker.artemis.livenessProbeInitialDelaySecond` 
could be replaced by `$args.componentConfig.livenessProbeInitialDelaySecond`, making it shorter.